src/main/java/com/afornalik/ox/controller/PlayerController.java

```diff
@@ -0,0 +1,56 @@
+package com.afornalik.ox.controller;
+
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.player.PlayerContainer;
+import com.afornalik.ox.view.UIOperations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class PlayerController {
+
+    private final UIOperations uiOperations;
+
+    PlayerController(UIOperations uiOperations) {
+        this.uiOperations = uiOperations;
+    }
+
+    String createPlayer() {
+
+        Map<String,Object> playerAttributes = new HashMap<>();
+
+        uiOperations.print("Player name : ");
+        playerAttributes.put("name",uiOperations.read());
+
+        do {
+            uiOperations.print("Select X or O mark ");
+            String tempSymbol = uiOperations.read();
+            if(tempSymbol.equalsIgnoreCase("x")){
+                playerAttributes.put("FieldStatus", FieldStatus.X);
+            }else if(tempSymbol.equalsIgnoreCase("o")){
+                playerAttributes.put("FieldStatus", FieldStatus.O);
+            }
+        }while(!playerAttributes.containsKey("FieldStatus"));
+
+        do {
+            uiOperations.print("Is this player make first move ? (Y/N) ");
```

Comment:
This one should be called only if there was no starting player selected before but I do not see any check for this anywhere.
Also, maybe instead of asking if player should start once it is created you could ask to select starting player once all of them are configured? This way above check would not be necessary.

Reply:
I changed it and now you only once declare player who start.